src/webui/components/Help/index.js

```diff
@@ -1,42 +1,54 @@
-import React from 'react';
-import SyntaxHighlighter, {registerLanguage} from 'react-syntax-highlighter/dist/light';
-import sunburst from 'react-syntax-highlighter/src/styles/sunburst';
-import js from 'react-syntax-highlighter/dist/languages/javascript';
-
-import classes from './help.scss';
-import {getRegistryURL} from '../../utils/url';
-
-registerLanguage('javascript', js);
-
-const Help = () => {
-  const registryURL = getRegistryURL();
-
-    return (
-      <div className={classes.help}>
-        <li className={classes.noPkg}>
-          <h1 className={classes.noPkgTitle}>
-            No Package Published Yet
-          </h1>
-          <div className={classes.noPkgIntro}>
-            <div>
-              To publish your first package just:
-            </div>
-            <br/>
-            <strong>
-              1. Login
-            </strong>
-            <SyntaxHighlighter language='javascript' style={sunburst} id="adduser">
-              {`npm adduser --registry  ${registryURL}`}
-            </SyntaxHighlighter>
-            <strong>2. Publish</strong>
-            <SyntaxHighlighter language='javascript' style={sunburst} id="publish">
-              {`npm publish --registry ${registryURL}`}
-            </SyntaxHighlighter>
-            <strong>3. Refresh this page!</strong>
-          </div>
-        </li>
-      </div>
-    );
+/**
+ * @prettier
+ * @flow
+ */
+
+import React, { Fragment } from 'react';
+import { CardStyled as Card } from './styles';
+import CardActions from '@material-ui/core/CardActions/index';
+import CardContent from '@material-ui/core/CardContent/index';
+import Button from '@material-ui/core/Button/index';
+import Typography from '@material-ui/core/Typography/index';
+
+import CopyToClipBoard from '../CopyToClipBoard/index';
+
+import { getRegistryURL } from '../../utils/url';
+
+import type { Node } from 'react';
```

Comment:
could you please move it to the top? "import React, { Fragment } from 'react'; import type { Node } from 'react';" Then we have reacts imports together :)